packages/matter.js/src/cluster/definitions/OperationalCredentialsCluster.ts

```diff
@@ -0,0 +1,780 @@
+         *
+         * The number of entries in this list shall match the number of entries in the NOCs attribute.
+         *
+         * @see {@link MatterCoreSpecificationV1_1} § 11.17.5.2
+         */
+        fabrics: FabricScopedAttribute(1, TlvArray(TlvFabricDescriptorStruct), { persistent: true, default: [] }),
+
+        /**
+         * This attribute contains the number of Fabrics that are supported by the device. This value is fixed for a
+         * particular device.
+         *
+         * @see {@link MatterCoreSpecificationV1_1} § 11.17.5.3
+         */
+        supportedFabrics: FixedAttribute(2, TlvUInt8.bound({ min: 5, max: 254 })),
+
+        /**
+         * This attribute contains the number of Fabrics to which the device is currently commissioned. This attribute
+         * shall be equal to the following:
+         *
+         *   • The number of entries in the NOCs attribute.
+         *
+         *   • The number of entries in the Fabrics attribute.
+         *
+         * Upon Factory Data Reset, this attribute shall be set to a default value of 0.
+         *
+         * @see {@link MatterCoreSpecificationV1_1} § 11.17.5.4
+         */
+        commissionedFabrics: Attribute(3, TlvUInt8, { persistent: true }),
+
+        /**
+         * This attribute shall contain a read-only list of Trusted Root CA Certificates installed on the Node, as
+         * octet strings containing their Matter Certificate Encoding representation.
+         *
+         * These certificates are installed through the AddTrustedRootCertificate command.
+         *
+         * Depending on the method of storage employed by the server, either shared storage for identical root
+         * certificates shared by many fabrics, or individually stored root certificate per fabric, multiple
+         *
+         * identical root certificates MAY legally appear within the list.
+         *
+         * To match a root with a given fabric, the root certificate’s subject and subject public key need to be
+         * cross-referenced with the NOC or ICAC certificates that appear in the NOCs attribute for a given fabric.
+         *
+         * Upon Factory Data Reset, this attribute shall be set to a default value whereby the list is empty.
+         *
+         * @see {@link MatterCoreSpecificationV1_1} § 11.17.5.5
+         */
+        trustedRootCertificates: Attribute(4, TlvArray(TlvByteString), { persistent: true, omitChanges: true, default: [] }),
+
+        /**
+         * This attribute shall contain accessing fabric index.
+         *
+         * This attribute is useful to contextualize Fabric-Scoped entries obtained from response commands or attribute
+         * reads, since a given Fabric may be referenced by a different Fabric Index locally on a remote Node.
+         *
+         * @see {@link MatterCoreSpecificationV1_1} § 11.17.5.6
+         */
+        currentFabricIndex: Attribute(5, TlvUInt8, { default: 0 })
```

Comment:
FabricIndex

Reply:
Spec specifies as uint8 but I think it's safe to assume this is a bug.  I overrode and left a comment documenting